Camisa.java

```diff
@@ -0,0 +1,162 @@
+package br.unit.modulo4.entidades;
+
+import java.util.Random;
+
+//Essa classe serve para simular um objeto do tipo "Camisa". Porém aqui, ao invés de o construtor retornar um objeto, retorna uma String com a cor da camisa para que possa ser armazenada na Lista.

```

Comment:
Entendi. Não é preciso nem instanciá-la né?

Reply:
Exatemente. Até comecei criando como um objeto mesmo. Porém aí precisariamos alterar a estrutura de armazenamento. Se isso for possível, ficaria até mais interessante.

Reply:
Entendi. Achei que cumpre bem seu propósito, mas nada impede de avançar. Vai depender do tempo. Estou tendo problemas com GitHub aqui pra colocar o meu branch, tá dando vários erros 500.

Reply:
Aqui também está bem ruim. Deve ser o servidor deles.